s3repo/model.py

```diff
@@ -135,17 +143,14 @@ def _upload_files(self, package, tarantool_series, origin_files):
             dist_path_list.insert(0, self.s3_settings['base_path'])
 
         dist_path = '/'.join(dist_path_list)
+        dist_base = self.get_supported_repos()['distrs'][package.dist]['base']
 
         # List of repositories where the new package has been uploaded,
         # but the metainformation hasn't been updated yet.
         unsync_repos_local = set()
         for filename, file in package.files.items():
             repo_path, path = S3AsyncModel._format_paths(dist_path,
-                package.dist_version, filename)
-
-            if repo_path is None:
-                raise RuntimeError('Can\'t to form the path to upload the file "' +
-                    filename + '".')
+                package.dist_version, dist_base, filename)
```

Comment:
Nit: I suggest to run flake8 before pushing and, if you'll have a free time, add it into CI.

> ./s3repo/model.py:153:17: E128 continuation line under-indented for visual indent

Feel free to ignore now, just noted for future.

Reply:
Hmm, I used `pylint` and it skeep this warning. I'll update this piece of code.